framework/src/Volo.Abp.IdentityModel/Volo/Abp/IdentityModel/IdentityModelAuthenticationService.cs

```diff
@@ -125,6 +129,8 @@ private IdentityClientConfiguration GetClientConfiguration(string identityClient
         {
             using (var httpClient = HttpClientFactory.CreateClient())
             {
+                AddHeaders(httpClient);
```

Comment:
`httpClient `should be named so that we can configure it.  
After that we can reuse `httpClient` in `GetDiscoveryResponse `and `GetTokenResponse `methods

Reply:
Yes, we can